bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/classificationstore.js

```diff
@@ -77,6 +76,14 @@ pimcore.object.classes.data.classificationstore = Class.create(pimcore.object.cl
             value: this.datax.allowedGroupIds
         });
 
+        this.specificPanel.add({
+            xtype: "numberfield",
+            fieldLabel: t("maximum_items"),
```

Comment:
can we use a dedicated label here so that it is clear to the user that it is about groups

Reply:
Yes sure, but I can only translate the English and German text.

Is there already an idea for translating them?

Reply:
Note that only en.json has to be touched. The other languages are managed in POEditor.
See https://pimcore.com/docs/6.x/Development_Documentation/Multi_Language_i18n/Admin_Translations.html

Reply:
Done ;-)

The linked "POEditor" link in the documentation is dead -> https://pimcore.com/en/resources/translations

No idea what the new / correct link is otherwise I would adjust it. If necessary you take it up as a to-do?

Reply:
thx for the notice, I fixed the link in https://github.com/pimcore/pimcore/commit/df66f7a13d050ba198108920b3ddf081029bbb7b